play.pokemonshowdown.com/src/panel-popups.tsx

```diff
@@ -1543,7 +1543,8 @@ class BattleOptionsPanel extends PSRoomPanel {
 	override render() {
 		const room = this.props.room;
 		const battleRoom = this.getBattleRoom();
-
+		const isPlayer = ['\u2606', '\u2605'].includes(battleRoom!.users[PS.user.userid]?.charAt(0));
+		const canOfferTie = (battleRoom!.battle.turn >= 100 && isPlayer) || PS.user.group === '~';
```

Comment:
This would crash if you tried to open the battle options popup in Main Menu or the challenge screen.